Dockerfile

```diff
@@ -42,4 +45,7 @@ ADD https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/${
 ENV JAVA_OPTS "-Dlog4j.configurationFile=${CATALINA_HOME}/conf/log4j2.xml"
 ADD src/docker/start.sh                         /docker/
 RUN chmod +x                                    /docker/start.sh
+RUN chown -R tomcat /docker/
+RUN chown -R tomcat ${CATALINA_HOME}/conf/
```

Comment:
You could merge line 48 and 49 to this:
`RUN chown -R tomcat /docker/ ${CATALINA_HOME}/conf/`

This would reduce the created layers. Unsure about line 22. Is access for user tomcat to /usr/local/tomcat needed before line 48? Up till line 50, all actions will be executed as root so i think it should also be includeable?

Reply:
Okay, I have rewritten it according to your suggestions and the image builds just fine.